Content.Server/Chemistry/EntitySystems/MedipenRefillerSystem.cs

```diff
@@ -0,0 +1,237 @@
+
+using Content.Server.Chemistry.Components;
+using Content.Server.Chemistry.Containers.EntitySystems;
+using Content.Server.UserInterface;
+using Content.Shared.Chemistry;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.Coordinates;
+using Content.Shared.Emag.Components;
+using Content.Shared.Emag.Systems;
+using Content.Shared.FixedPoint;
+using Robust.Server.Audio;
+using Robust.Server.GameObjects;
+using Robust.Shared.Containers;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chemistry.EntitySystems;
+
+public sealed class MedipenRefillerSystem : EntitySystem
```

Comment:
same with client inherit shared refiller system and remove this ones emag handler